src/numeric/impl_numeric.rs

```diff
@@ -46,6 +46,84 @@ impl<A, S, D> ArrayBase<S, D>
         sum
     }
 
+    /// Return a reference to a maximum of all values.
+    /// Return None if a comparison fails or if self is empty.
+    /// 
+    /// # Example
+    /// ```
+    /// use ndarray::arr2;
+    /// use std::f64;
+    /// 
+    /// let a = arr2(&[[1., 2.], [3., 4.]]);
+    /// 
+    /// assert_eq!(a.scalar_max(), Some(&4.));
+    /// 
+    /// let b = arr2(&[[1., f64::NAN], [3., 4.]]);
+    /// 
+    /// assert_eq!(b.scalar_max(), None);
+    /// ```
+    pub fn scalar_max(&self) -> Option<&A>
```

Comment:
Any reason this is not named max?

Reply:
Last I knew the plan was to rename scalar_sum to sum, so don't want it to be followed. That said, last I knew was some time ago. Commit b75d438daea2d0c5ce684fed1a5a1ebde7d4e166

Reply:
Excellent, renamed!